src/Response/CsvResponse.php

```diff
@@ -30,7 +30,7 @@ class CsvResponse implements ResponseInterface
     /**
      * The response CSV parsed into an array (each CSV line an array element).
      *
-     * @var array
+     * @var string[]
      */
     protected $response;
```

Comment:
This attribute isn't referenced anywhere, it can be deleted safely. I'll create an issue.